pkg/ndctl/ndctl.go

```diff
@@ -62,14 +66,18 @@ func (ctx *Context) GetBuses() []*Bus {
 
 //CreateNamespace create new namespace with given opts
 func (ctx *Context) CreateNamespace(opts CreateNamespaceOpts) (*Namespace, error) {
+	var err error
+	var ns *Namespace
 	for _, bus := range ctx.GetBuses() {
 		for _, r := range bus.ActiveRegions() {
-			if ns, err := r.CreateNamespace(opts); err == nil {
+			if ns, err = r.CreateNamespace(opts); err == nil {
 				return ns, nil
+			} else {
+				err = errors.Wrap(err, "")
 			}
 		}
 	}
-	return nil, fmt.Errorf("Failed to create namespace")
+	return nil, errors.Wrap(err, "Failed to create namespace")
```

Comment:
It's expected that you only get the last error, with "Failed to create namespace" as prefix. 

If you want to collect all errors, then you need to collect them in a list and then somehow concatenate. If it fails the same way for all regions, then that's probably overkill and you can simply clarify that you return the latest error with "failed to create namespace in all regions, last error is".

BTW, error strings should always start lowercase. That's because another prefix might be added higher up, and then mixed case looks weird.

Reply:
ah yes, silly me, I tried to "optimize" num of vars there... we need more variables. 
Same from all regions: I was thinking same way, but it feels bit overkill to start analyzing error msg contents in that function